<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtValveControlWidget</class>
 <widget class="QWidget" name="QtValveControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="valveGroupBox">
     <property name="title">
      <string>Valve 1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>14</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="desiredPortLabel">
        <property name="text">
         <string>Desired Port</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2" colspan="2">
       <widget class="QComboBox" name="desiredPortComboBox">
        <item>
         <property name="text">
          <string>Position</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="valveConfigurationLabel">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Valve Configuration</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4" rowspan="2">
       <widget class="QPushButton" name="changePortButton">
        <property name="text">
         <string>Change Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLabel" name="rotationDirectionLabel">
        <property name="text">
         <string>Rotation Direction</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="2" colspan="2">
       <widget class="QComboBox" name="desiredRotationComboBox">
        <item>
         <property name="text">
          <string>Clockwise</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Counter Clockwise</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="valveStatusLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Valve Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>desiredPortComboBox</tabstop>
  <tabstop>desiredRotationComboBox</tabstop>
  <tabstop>changePortButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
